checkstyle.xml

```diff
@@ -17,6 +17,11 @@
  -->
 
 <module name="Checker">
+
+  <!-- Reference the suppression file -->
+  <module name="SuppressionFilter">
+    <property name="file" value="suppression.xml"/>
+  </module>
```

Comment:
there is already checkstyle-suppressions.xml defined in this file. Why not to use the name